src/pages/docs/endpoints/examples.mdx

```diff
@@ -53,6 +53,55 @@ Get examples from dictionary
 
  ***
 
+ **Get Poverbs**
```

Comment:
remove all these lines and update the markdown typo in line 32

Reply:
Should I also remove the lines for the response of proverbs?